app/controllers/organisation/consultations_controller.rb

```diff
@@ -96,6 +117,23 @@ def publish
 		redirect_back fallback_location: root_path,  flash_success_info: "Consultation was successfully approved."
 	end
 
+  def extend_deadline
+    @consultation.update(secure_params)
+    @consultation.publish
+    redirect_back fallback_location: root_path,  flash_success_info: "Consultation deadline was extended and successfully published."
+  end
+
+  def make_changes
+    @consultation.create_response_round
+    @consultation.submitted!
+    redirect_to edit_english_summary_organisation_consultation_path(@consultation)
+  end
+
+  def invite_respondents
+    Respondent.invite_respondent(@consultation, @organisation, params)
```

Comment:
@BALAJI-BALA - do not pass all params into the model, you must pass respondent_ids, emails one by one.

The model should not deal with controller params